### PR TITLE
Lesson 5. Array listing

### PR DESCRIPTION
**Pull request by snoozee**

### CODE REVIEW

File: src/main/java/collections/OccurrenceCounter.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        for(String word : words) {
+            if (word.equals(target)) {
+                count++;
+            }
+        }
+        return count;
+    }
+
+    public static void main(String[] args) {
+        List<String> wordList = List.of("Bradley", "Leopard-1", "Leopard-2", "T-72", "Bradley", "T-90");
```
Comment (SerhiiPasko) [inline src/main/java/collections/OccurrenceCounter.java:18]: Good names

**Review by SerhiiPasko — CHANGES_REQUESTED** [review]
Hi. Currently it's only a half of homework

File: src/main/java/phonebook/TelephoneDirectory.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package phonebook;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TelephoneDirectory {
+    private List<Record> phoneBook;
```
Comment (SerhiiPasko) [inline src/main/java/phonebook/TelephoneDirectory.java]: `= new  ArrayList<>();`

In this case you cannot write a constructor

File: src/main/java/phonebook/Record.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package phonebook;
+
+public class Record {
```
Comment (SerhiiPasko) [inline src/main/java/phonebook/Record.java]: `record` is a keyword in java. If possible, better to use synonims

**Review by SerhiiPasko — COMMENTED** [review]
Cool

File: src/main/java/collections/OccurrenceCounter.java (hunk 1 above)
Comment (snoozee) [reply src/main/java/collections/OccurrenceCounter.java:18]: Thx

**Review by SerhiiPasko — APPROVED** [review]
Cool

**Pull request merged**